_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -141,3 +141,14 @@ Development. Where all the magic which keeps this app running happens. It saw so
 - Some deprecations as well as a lot of code cleanup happened. ([@TacoTheDank](https://github.com/TacoTheDank))
 
 - The code now prevents the YouTube website from accessing the available Java packages from NewPipe's parser. This removes a possible attack surface. ([@wb9688](https://github.com/wb9688))
+
+## Where to get this brand-new version
+
+NewPipe notifies you about new versions, you can download them when you press the notification. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository. If you have problems installing you may need to uninstall NewPipe then reinstall (make sure to backup data).
```

Comment:
```suggestion
NewPipe notifies you about new versions. You can download them when you press the notification. An alternative is the [GitHub release page](https://github.com/TeamNewPipe/NewPipe/releases). If you use the [F-Droid app](https://f-droid.org/), it notifies you as well about an update for [NewPipe](https://f-droid.org/packages/org.schabi.newpipe/) - please keep in mind that it can take F-Droid a while to update their repository. If you have problems installing you may need to uninstall NewPipe then reinstall (make sure to backup data).
```